demos/fleet_provisioning/fleet_provisioning_with_csr/demo_template.json

```diff
@@ -0,0 +1,52 @@
+ {
+   "Parameters": {
+     "SerialNumber": {
+       "Type": "String"
+     },
+     "AWS::IoT::Certificate::Id": {
+       "Type": "String"
+     }
+   },
+   "Resources": {
+     "certificate": {
+       "Properties": {
+         "CertificateId": {
+           "Ref": "AWS::IoT::Certificate::Id"
+         },
+         "Status": "Active"
+       },
+       "Type": "AWS::IoT::Certificate"
+     },
+     "policy": {
+       "Properties": {
+         "PolicyName": "demo_thing_policy"
+       },
+       "Type": "AWS::IoT::Policy"
+     },
+     "thing": {
+       "OverrideSettings": {
+         "AttributePayload": "MERGE",
+         "ThingGroups": "DO_NOTHING",
+         "ThingTypeName": "REPLACE"
+       },
+       "Properties": {
+         "AttributePayload": {},
+         "ThingGroups": [],
+         "ThingName": {
```

Comment:
A comment can be added here that the `Fn::Join` created the thing name as the concatenation of `fp_demo_` and the serial number sent by the client device (or demo) as an attribute to AWS IoT Core

Reply:
Can't add comments to json files

Reply:
Okay, it's fine then, maybe, you can mention this aspect in the comment within `demo_config.h` for the template macro